File: oc-demo-project-pod/oc-demo-project-pod/ViewController.m
//
//  ViewController.m
//  oc-demo-project-pod
//
//  Created by kingcos on 2019/11/1.
//  Copyright © 2019 kingcos. All rights reserved.
//

#import "ViewController.h"
#import <oc-pod/ObjC_Foo.h>

@import swift_pod;

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
    
    [[[ObjC_Foo alloc] init] foo];
    
    [[[Swift_Foo alloc] init] foo];
    NSLog(@"%@", [[[Swift_Foo alloc] init] strings]);
}


@end

File: swift-demo-project-pod-use_frameworks/swift-demo-project-pod/ViewController.swift
//
//  ViewController.swift
//  swift-demo-project-pod
//
//  Created by kingcos on 2019/11/4.
//  Copyright © 2019 kingcos. All rights reserved.
//

import UIKit
import swift_pod
import oc_pod

class ViewController: UIViewController {
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
        
        ObjC_Foo().foo()
        
//        Swift_Foo().foo()
        print(Swift_Foo().strings())
    }
}
